### ISSUE TITLE
Saves not showing up with SDUSB installed

### ISSUE THREAD

**Issue by OldGnashburg** [Xpl0itU/savemii#57]
So I installed the ISFSHax Wafel plugin SD USB and transfered everything from my HDD to a partition on a 1TB microSD all my games show up, they run off the home menu like a regular USB device. But when I tried using this homebrew none of my saves are showing up when they previously did. There should be no SD access issues on the SD USB end so I'm just curious what's going on.

**Comment by w3irDv** [comment]
Hi @OldGnashburg, if you have still the problem, I'll try to reproduce it, but first I'll need to setup SDUSB. Please can you confirm is savemii works or not in this scenario? Thanks!

**Comment by w3irDv** [comment]
Hi @OldGnashburg, I have not been able to reproduce this problem. In my case, (new) savemii works with (last) SDUSB the same way that it does with a HDD. I have installed some of my games and transferred without problems the backups I previously have. When do you say that the saves are not shown, do you mean they are not shown in the wiiu title list menu, or that the restore slots appears empty?

**Comment by OldGnashburg** [comment]
The restore slots were empty and I don't think I was able to make backups either. I did a fresh install of SDUSB and the partition because I was having the same issue with the USBPart plugin and now it all works. Only weird thing is some saves still wouldn't show up even though they were on the SD and everything was correct. I had to make a backup first and then swap the files out in order for some of them to show up. It was weird. I have no idea what happened or why things mostly work now but not going to question it.
